playwright-tests/e2e/tests/comments.spec.ts

```diff
@@ -0,0 +1,102 @@
+//comments-spec.ts
+import { Browser, Page, expect } from "@playwright/test";
+import { test } from "../fixtures/index";
+import { Users } from "../../constants";
+import { TaskPage } from "../pom/tasks";
+import { Auth } from "../pom/login";
+import { faker } from "@faker-js/faker";
+import LoginPage from "../pom/login";
+
+test.describe("Comment Page", () => {
+  let todoName: string;
+  let assigneeComment: string;
+  let assignerComment: string;
+
+  test.beforeEach(({ page }) => {
+    todoName = faker.word.verb(5);
+    assigneeComment = faker.lorem.sentence();
+    assignerComment = faker.lorem.sentence();
+  });
+
+  test("comment on task as creator and assigned user", async ({
+    page,
+    loginPage,
+    browser,
+    taskPage,
+  }) => {
+    page.goto("/");
+
+    await test.step("step 1 - create a new task by assigner", () =>
+      taskPage.createTaskAndVerify({
+        taskName: todoName,
+        userName: "Sam Smith",
+      }));
+
+    await test.step("step 2 - go to that newely created task", async () => {
+      await page.getByTestId("tasks-pending-table").getByText(todoName).click();
+
+      const endPoint = todoName.replace(/\s+/g, "-");
+
+      await page.waitForResponse(response => response.url().includes(endPoint));
+      expect(await page.locator("h1")).toHaveText(todoName);
```

Comment:
`.locator` is not a promise. You can avoid the `await` here.
On the other hand, `toHaveText` is an auto-retrying assertion. So please make this:
```suggestion
      await expect(page.locator("h1")).toHaveText(todoName);
```